huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go

```diff
@@ -195,6 +195,13 @@ func ResourceCCENodePool() *schema.Resource {
 					}},
 			},
 			"tags": common.TagsSchema(),
+			// charge info: charging_mode, period_unit, period, auto_renew, auto_pay
+			"charging_mode": common.SchemaChargingMode(nil),
+			"period_unit":   common.SchemaPeriodUnit(nil),
+			"period":        common.SchemaPeriod(nil),
+			"auto_renew":    common.SchemaAutoRenew(nil),
```

Comment:
Can we update the auto-renew via public function?

Reply:
the resource id is node id, which is not easy to get
and we can't update auto_renew for every node